wiki/Enhancement_Requests.md

```diff
@@ -282,7 +282,7 @@ would only consist of gap characters, and be virtual).
 Should there be a better way of doing this?
 
 Related to CO lines, [EMBLs AS
-lines](http://www.ebi.ac.uk/embl/Documentation/User_manual/usrman.html#3_4_13)
+lines]<http://ftp.ebi.ac.uk/pub/databases/embl/doc/usrman.txt#3_4_13>
```

Comment:
These should be round brackets not the less than/greater than signs (for markdown links).